.github/workflows/pre-commit.yml

```
@@ -14,11 +14,13 @@ jobs:
   pre-commit:
     runs-on: ubuntu-22.04
     steps:
-      - name: Checkout code
+      - name: Checkout code (PR branch)
         uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)